frontends/ast/simplify.cc

```diff
@@ -2904,7 +2904,12 @@ AstNode *AstNode::readmem(bool is_readmemh, std::string mem_filename, AstNode *m
 	std::ifstream f;
 	f.open(mem_filename.c_str());
 	if (f.fail()) {
-		std::string path = filename.substr(0, filename.find_last_of("\\/")+1);
+#ifdef _WIN32
+		char slash = '\\';
```

Comment:
This is actually incorrect, since on Windows you can use both slashes, and this is useful in portable code.